composer.json

```diff
@@ -7,30 +7,28 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.3",
-        "drupal/components": "^2.4",
         "drupal/core": "^8.9 || ^9.1",
-        "drupal/ui_patterns": "^1.0",
-        "drupal/ui_patterns_settings": "^1.0",
-        "openeuropa/oe_bootstrap_theme": "1.0.0-alpha4"
+        "openeuropa/oe_bootstrap_theme": "^1.0.0-alpha5"
     },
     "require-dev": {
         "composer/installers": "^1.11",
-        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
-        "drupal/config_devel": "~1.2",
         "drupal/better_exposed_filters": "^5.0@beta",
+        "drupal/config_devel": "~1.2",
+        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
+        "drupal/ctools": "^3.4",
```

Comment:
something is forcing drupal 9 even in composer lowest, I think that is the actual source of this core_version_requirement issue